README.md

```diff
@@ -65,6 +65,23 @@ When invoking these commands, you can combine the short versions into a single f
 % secureli init -ry
 ```
 
+# Usage Statistic Observability
+
+Supported observability platform.
```

Comment:
I think this should be changed to more of a "tutorial" or sorts using New Relic as an example and mention that if other backends should be supported, to create an issue or even better, contribute to project.